libraries/vault_installation_git.rb

```diff
@@ -4,90 +4,61 @@
 #
 # Copyright 2015-2016, Bloomberg Finance L.P.
 #
-require 'poise'
+# A `vault_installation` provider which manages Vault installation
+# from the Go source builds.
+#
 
-module VaultCookbook
-  module Provider
-    # A `vault_installation` provider which manages Vault installation
-    # from the Go source builds.
-    # @action create
-    # @action remove
-    # @provides vault_installation
-    # @example
-    #   vault_installation '0.5.0' do
-    #     provider 'git'
-    #   end
-    # @since 2.0
-    class VaultInstallationGit < Chef::Provider
-      include Poise(inversion: :vault_installation)
-      provides(:git)
-      inversion_attribute('hashicorp-vault')
+resource_name :vault_install_git
```

Comment:
Should we keep the resource names consistent? The previous resource name is `vault_installation_git`.